python/lsst/afw/table/arrays.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from ._arrays import *
```

Comment:
I hope we can avoid it because we lose so much readability and linter functionality with this construct. How bad would it be to import symbols by name or use the library prefix? Obviously I won't fully appreciate the utility until I do some pybind11 wrapping myself.